### PR TITLE
Issues/#211 additional vocab

### PR DESCRIPTION
**Pull request by barthanssens**
This PR addresses GitHub issue: #211 

Briefly describe the changes proposed in this PR:

- Added VOID, DCAT, ORG and ROV vocabularies

I guess the formatting should be OK (I'm using Netbeans, not Eclipse)

Make sure you've followed the [Contributor Guidelines](https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md). In particular (please tick to indicate you've taken care of it):

- [] RDF4J code formatting has been applied
- [] tests are included
- [] all tests succeed

### CODE REVIEW

**Comment by jeenbroekstra** [comment]
Looks good to me, thanks! Will merge ASAP.

File: core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DCAT.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```
Comment (jeenbroekstra) [inline core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DCAT.java]: Very minor this, but this is not the correct copyright header for new contributions. The correct header is as follows:

`/**
 * Copyright (c) 2015 Eclipse RDF4J contributors, and others.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Distribution License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/org/documents/edl-v10.php.
 */`

(basically the mention of "Aduna" shouldn't be there for new code).

Comment (barthanssens) [inline core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DCAT.java]: Ok, thanks, I'll change the copyright header(s)

**Comment by jeenbroekstra** [comment]
Approved! Thanks again 👍

**Pull request merged**